LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2017, Project OSRM contributors
```

Comment:
technically we don't need this, we're just using it as a dependency, we don't re-distribute it.

let me add a license via Github, same as osrm though.

Reply:
Ok done. You can remove this one

Reply:
Okay, I've removed it!